docs/topics/mta-rn-known-issues-6-2-0.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-6-2-0_{context}"]
+= Known issues
+
+MTA version 6.2.0 has the following issues.
+
+.MTA fails to run an analysis of private GitHub repos
+
+Analysis of private GitHub repositories fails. link:https://issues.redhat.com/browse/MTA-810[MTA-810]
+
+.Analysis fails when fetching rules from a repository with a folder that contains spaces in its name
```

Comment:
This bug is not resolved and from Jeff comment on the bug, it can't be:

> Spaces in directories specified in --userRulesDirectory cannot be supported by windup because --userRulesDirectory accepts a space-delimited list of directory paths.



 I asked them to add a little tooltip to indicate that, but this won't be included in this version.

Reply:
@abrugaro - i have added your additional line for further information but it is your call if you removed. 

It is caught in the Jira filter for known issues, and we could expand the ticket description.